src/app/models/schedule.model.ts

```diff
@@ -42,7 +42,8 @@ export class Schedule {
 
     this.height = 600;
 
-    this.colors = ['#ffd4df', '#ceeffc', '#fff4d0', '#dcf7da', '#f7e2f7','#ffd4df', '#ceeffc', '#fff4d0', '#dcf7da', '#f7e2f7','#ffd4df', '#ceeffc'];
+    
+    this.colors = ['#ffd4df', '#ceeffc', '#fff4d0', '#dcf7da', '#f7e2f7', '#ede6df', '#ffe9cf'];
```

Comment:
Can we actually take this opportunity to make these class constants, since that's what they really should be